css/build.css

```diff
@@ -122,19 +122,19 @@
   to use UNICODE icons instead of SVG icons.
 */
 
-.fa-times-thin {
+.close-icon .fa-times-thin {
```

Comment:
I think we can be more specific than this.

`[data-widget-package="com.fliplet.dynamic-lists"]` would be a pretty good selector to add so that it only affects `.fa-times-thin` within LFDs.

Reply:
@tonytlwu  Agree.
Added data attribute to the selector for a more specified selector.